diesel/src/expression_methods/global_expression_methods.rs

```diff
@@ -9,6 +9,9 @@ use crate::sql_types::{SingleValue, SqlType};
 pub trait ExpressionMethods: Expression + Sized {
     /// Creates a SQL `=` expression.
     ///
+    /// Note that this function follows SQL semantics around `None`/`null` values,
```

Comment:
It might be useful to link the sqlite specific [`IS`](https://docs.diesel.rs/master/diesel/expression_methods/trait.SqliteExpressionMethods.html#method.is) operator and the postgres specific [`IS NOT DISTINCT FROM`](https://docs.diesel.rs/master/diesel/expression_methods/trait.PgExpressionMethods.html#method.is_not_distinct_from) here, as both are more in line with rusts `=` operator.

Reply:
Great suggestion!